### DISCUSSION TITLE
a2l.py generated from jsonSchema misses out the class Asap2File

### DISCUSSION THREAD

**Discussion by ramkrganesh** [Luncher91/A2LParser#41] [Q&A]
Hi Andreas,

When I try to generate a2l.py file using jsonSchema, it creates a class called `class Model(BaseModel)` instead of `class Asap2File(BaseModel)`

But, the a2l.py file from "Assets" of v2.6.3 contains the expected Asap2File class. Could you please try to re-generate a2l.py using v2.6.3 and see if you could also re-create this problem ?

Note: I use "datamodel-codegen" version 0.25.2

BR,
Ram

**Comment by Luncher91** [comment]
Hi Ram,

I cannot reproduce the issue.
Can you check your schema file if "Model" has been defined there?
If so, then it has probably something to do with the version of com.fasterxml.jackson you have used. The pom file specifies 2.15.2
Otherwise it maybe is related to the used Python version. I used Python 3.11 for the generation of the release files and 3.9 locally.

Kind regards,
Andreas
